files/galaxy/config/file_source_templates.yml

```diff
@@ -0,0 +1,5 @@
+- include: ./lib/galaxy/files/templates/examples/production_azure.yml
```

Comment:
we need to decide what we do with those templates ...
Do we copy them into a specific location? How do we handle version updates to those files then? I don't think we need to template them out, but we need to decide where to store them.

Reply:
a few thoughts:

* we might want to add new templates, and those are then most likely not in the Galaxy source tree
* upstream might add updates - a new version of the file-source with a new version number
  *  those we could sync like we sync now other config files
* I assume those templates need to be accessible by celery and other services - so on a shared filesystem

Maybe a separate folder under mutable-config/file-source-templates and mutable-config/object-store-templates/ ?

Reply:
@jmchilton any thoughts?

Reply:
I think that is a good solution. We can have a separate directory and append it to the lists [here](https://github.com/usegalaxy-eu/infrastructure-playbook/blob/master/group_vars/sn06.yml#L378-L469). We can add a couple of variables there and let the `ansible-galaxy` role copy everything and handle these files.